client/components/QandA/QAComponents/ReportButton.jsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
+import axios from 'axios';
 
-const ReportButton = (props) => {
+const ReportButton =({id, path}) => {
+
+  const updateReport = (targetId) => {
+    axios.put(path, {
+      params: targetId,
+    })
+    .then(response => response)
+    .catch(err => err)
```

Comment:
We can go through and change these later if needed but will this catch statement do anything with the error? 
Should we be throw new error(err) or console.log(err) instead?